modules/storages/spec/requests/api/v3/file_links/file_links_spec.rb

```diff
@@ -91,6 +91,76 @@
     login_as current_user
   end
 
+  describe 'POST /api/v3/file_links' do
+    let(:path) { '/api/v3/file_links' }
+    let(:permissions) { %i(manage_file_links) }
+    let(:storage_url1) { storage.host }
+    let(:storage_url2) { storage.host }
```

Comment:
No need to use 2 times the same storage host in different variables, or do I miss something?

Reply:
Im not sure. I will check it.